ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -112,6 +112,90 @@ def test003_cancel_button_edit_departments(self, save):
                                                                                       order_departments))
             self.assertEqual(current_departments, order_departments)
 
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test006_update_material_type(self, save):
```

Comment:
add new test cases to the end of the file

Reply:
What do you mean by that ?

Reply:
What do you mean by that ?

Reply:
new test cases should be added to the end of the tests file not in the middle of the code.

it should go after this line
![image](https://user-images.githubusercontent.com/17981089/61049511-f6ccd200-a3e4-11e9-8fe9-d4bcd370acb0.png)

Reply:
Done